pkg/types/azure/platform.go

```diff
@@ -59,6 +59,11 @@ type Platform struct {
 	// +optional
 	VirtualNetwork string `json:"virtualNetwork,omitempty"`
 
+	// InternalLBIP specifies the IP address to be used for the Internal Load Balancer.
+	//
+	// +optional
+	InternalLBIP string `json:"internalLBIP,omitempty"`
```

Comment:
I don't see a need to expose this as a field in the install config. We should be able to manage this well enough for users with the info we already have (i.e. `machineCidr`). If users want to specify a specific ip address, which should be a rare occurrence, they can edit through manifests. Keeping this simple by leaving this out of the install config would be best for users.